src/modules/45-dashboards/components/DashboardList/DashboardList.tsx

```diff
@@ -36,6 +35,15 @@ const DashboardList: React.FC<DashboardListProps> = ({
 }): React.ReactElement => {
   const { getString } = useStrings()
   const { accountId, folderId } = useParams<{ accountId: string; folderId: string }>()
+  const { editableFolders } = useDashboardsContext()
+
+  const isCloneable: boolean = React.useMemo(() => {
+    return editableFolders ? editableFolders.length > 0 : false
+  }, [editableFolders])
+
+  const isDashboardEditable = (dashboard: DashboardModel): boolean => {
+    return editableFolders?.some(folder => folder.id === dashboard.folder.id) ?? false
```

Comment:
Might as well simplify
```suggestion
    return !!editableFolders?.some(folder => folder.id === dashboard.folder.id)
```